src/lib/server/api/discord.ts

```diff
@@ -46,6 +48,17 @@ async function createMessage(logger: Logger, channelId: Snowflake, data: CreateM
     return code;
 }
 
+export function constructAttachmentField(attachment: Attachment) {
+    const contentIdentifier = attachment.content_type?.split('/')[0] ?? 'file';
+    const attachmentInfo = attachment.url;
+
+    return {
+        name: `${contentIdentifier[0]?.toUpperCase().concat(contentIdentifier.substring(1))} Attachment`,
+        value: attachmentInfo,
+        inline: true,
+    };
+}
```

Comment:
Some array destructuring, non-null assertions, and empty-rest assertions would be great here.

Reply:
I've enforced expectations on `contentIdentifier` and `attachmentInfo` being strings here, along with there only being one `/` separator (i.e. `attachment.content_type?.split('/')`, if it exists is only two elements).